feat(logs): structlog configuration. (#86)
topsport-com-au/starlite-saqlalchemy: docs/logging.md

# Logging

> `starlite-saqlalchemy` has a baked-in structured logging pattern that facilitates the

Author: ```suggestion
`starlite-saqlalchemy` has structured logging baked-in, built around facilitating the
```
